README.md

```diff
@@ -8,6 +8,24 @@ follow the instructions:
 - `bundle:generate` - to generate default symfony bundles
 - `pimcore:generate:bundle` - to generate Pimcore bundles
 
+# Installation
+
+On your Pimcore X root project
+
+```bash
+$ composer -vvv require pimcore/bundle-generator
+```
+
+After that at `src/Kernel.php` on `registerBundlesToCollection` function register the bundle by using the following code snippet :
```

Comment:
the preferred way would be to go the standard pimcore way and use Pimcore extension manager or  `bin/console pimcore:bundle:enable` command for enabling the bundle... so there is no need to change the `Kernel` class...

Reply:
I have read the CLA Document and I hereby sign the CLA

Reply:
Yes,  Previously I did enable using command, but it is not working, and then I try to register it manually on Kernel class.